src/TestFSharp/TestFSharp.fsproj

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>ca8dab68-17e9-48bf-afa3-9ccfd4b0a3cd</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>TestFSharpGrains</RootNamespace>
+    <AssemblyName>TestFSharpGrains</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
```

Comment:
@jthelin - The reasons for referencing ```FSharp.Core, Version=4.4.0.0``` (which corresponds to F# 4.0) are that f# 4.0 is the current version of the F# runtime and also the version we are using in production (a bit selfish, I know :-) ). Additionally, I intend to add more tests - and some of them may end up being F# 4.0 specific. I would prefer to continue referencing F# 4.0 if at all possible.